cmd/cli/version/version.go

```diff
@@ -105,12 +109,67 @@ func (oV *VersionOptions) Run(ctx context.Context, cmd *cobra.Command) error {
 		serverVersion, err := util.GetAPIClient(ctx).Version(ctx)
 		if err != nil {
 			log.Ctx(ctx).Error().Err(err).Msgf("could not get server version")
-			return err
+			// Fail silently
+		} else {
+			versions.ServerVersion = serverVersion
+			columns = append(columns, serverVersionColumn)
 		}
+	}
 
-		versions.ServerVersion = serverVersion
-		columns = append(columns, serverVersionColumn)
+	// Assuming versions.ServerVersion may be nil if oV.ClientOnly is true
+	var serverVersion string
+	if versions.ServerVersion != nil {
+		serverVersion = versions.ServerVersion.GitVersion
 	}
 
+	checkForUpdates(ctx, versions.ClientVersion.GitVersion, serverVersion)
+
 	return output.OutputOne(cmd, columns, oV.OutputOpts, versions)
 }
+
+type ServerResponse struct {
+	Version string `json:"version"`
+	Message string `json:"message"`
+}
+
+func checkForUpdates(ctx context.Context, currentClientVersion, currentServerVersion string) {
+	// Construct URL with query parameters
+	u, err := url.Parse("http://update.bacalhau.org/version")
+	if err != nil {
+		log.Ctx(ctx).Error().Err(err).Msg("Failed to parse URL.")
+		return
+	}
+
+	q := u.Query()
+	q.Set("clientVersion", currentClientVersion)
+	if currentServerVersion != "" {
+		q.Set("serverVersion", currentServerVersion)
+	}
```

Comment:
Can we also add the other things in here? E.g. runtime.GOOS, runtime.GOARCH, config.GetUserId(), etc